pkg/cloud/aws/services/ec2/subnets.go

```diff
@@ -255,21 +304,26 @@ func (s *Service) deleteSubnet(id string) error {
 }
 
 func (s *Service) getSubnetTagParams(id string, public bool) tags.BuildParams {
-	var name strings.Builder
-
-	name.WriteString(s.scope.Name())
-	name.WriteString("-subnet-")
+	var routable string
 	if public {
-		name.WriteString("public")
+		routable = "public"
 	} else {
-		name.WriteString("private")
+		routable = "private"
 	}
 
+	var name strings.Builder
+	name.WriteString(s.scope.Name())
+	name.WriteString("-subnet-")
+	name.WriteString(routable)
+
 	return tags.BuildParams{
 		ClusterName: s.scope.Name(),
 		ResourceID:  id,
 		Lifecycle:   tags.ResourceLifecycleOwned,
 		Name:        aws.String(name.String()),
 		Role:        aws.String(tags.ValueCommonRole),
```

Comment:
As suggested, this Role could be private/public instead of "common", which I don't think it has any particular meaning referring to managed subnets.

/cc @detiber